### ISSUE TITLE
Crash when denied proxy access

### ISSUE THREAD

**Issue by rolandza** [notepad-plus-plus/notepad-plus-plus#847]
In our corp. environment we access internet through a TMG proxy server.
When launching np++ it tries to connect to the proxy (for update??) which in turn prompts for
user credentials. I don't want to enter my credentials at that stage so I click "Cancel" which I
have to do 2-3 times and then np++ crashes completely.

I have tried suggestions of renaming/removing the update folder, does not resolve the issue.
I have also edited config.xml and set "noUpdate" to "yes", does not resolve the issue.

I'm currently using np++ 6.8.2.0 (latest), but this behaviour was present on 6.8.0.0 and the version I used before that too.

If I enter my credentials for proxy access everything is OK, but I don't want to have to do this.
How do I disable this correctly and permanently?

![clipboard01masked](https://cloud.githubusercontent.com/assets/14104946/9652376/fc7597a2-521a-11e5-873a-b2742faba86f.png)

![clipboard02](https://cloud.githubusercontent.com/assets/14104946/9652339/9d91be96-521a-11e5-8f3e-80cecf29121f.png)

![clipboard03](https://cloud.githubusercontent.com/assets/14104946/9652342/a7cfbaa2-521a-11e5-859b-9ae1763a4d55.png)

I searched for a similar issue/thread and was not able to find anything, apologies in advance if this is a duplicate topic.

Roland

**Comment by rolandza** [comment]
I just found the proper procedure for crash analysis @ http://docs.notepad-plus-plus.org/index.php/Crashes

In following the prescribed procedure I have narrowed this issue down to the "PluginManager.dll" plugin.
Unfortunately the "Plugin Development Forum" referred to in the article is unavailable (404). Does anyone know who the author of this plugin is? It seems to be native to np++, is this correct?

I have removed the offending plugin which resolves my issue for now.

Roland
